globals.json

```diff
@@ -557,5 +557,72 @@
 		"YUI": false,
 		"Y": false,
 		"YUI_config": false
+	},
+	"shelljs": {
+		"target": false,
+		"echo": false,
+		"exit": false,
+		"cd": false,
+		"pwd": false,
+		"ls": false,
+		"find": false,
+		"cp": false,
+		"rm": false,
+		"mv": false,
+		"mkdir": false,
+		"test": false,
+		"cat": false,
+		"sed": false,
+		"grep": false,
+		"which": false,
+		"dirs": false,
+		"pushd": false,
+		"popd": false,
+		"env": false,
+		"exec": false,
+		"chmod": false,
+		"config": false,
+		"error": false,
+		"tempdir": false
```

Comment:
`shelljs` has a lot generic globals and using their global mode is generally frown upon in the Node world. I wouldn't add this just because it's a bad practise to use it.